run_tests.py

```diff
@@ -21,29 +21,31 @@ def wait_until_rc_is_ready():
     raise Exception("Remote controller failed to start.")
 
 
-if __name__ == "__main__":
-    with open("rc_stdout.log", "w") as rc_stdout:
-        with open("rc_stderr.log", "w") as rc_stderr:
-            rc_process = start_rc(rc_stdout, rc_stderr)
-            try:
-                wait_until_rc_is_ready()
-                sys.stdout.flush()
-
-                args = [
-                    "pytest",
-                    "--cov=hazelcast",
-                    "--cov-report=xml",
-                ]
-
-                enterprise_key = os.environ.get("HAZELCAST_ENTERPRISE_KEY", None)
-                if not enterprise_key:
-                    args.extend(["-m", "not enterprise"])
-
-                process = subprocess.run(args)
-                rc_process.kill()
-                rc_process.wait()
-                sys.exit(process.returncode)
-            except:
-                rc_process.kill()
-                rc_process.wait()
-                raise
+with open("rc_stdout.log", "w") as rc_stdout:
+    with open("rc_stderr.log", "w") as rc_stderr:
+        rc_process = start_rc(rc_stdout, rc_stderr)
+        try:
+            wait_until_rc_is_ready()
+            sys.stdout.flush()
+
+            args = [
+                "pytest",
+                "--cov=hazelcast",
+                "--cov-report=xml",
+            ]
+            args.extend(sys.argv[1:])
```

Comment:
Why did you add this line?

Reply:
To make arguments passed to `run_tests` passed to pytest, like:
```
./run_tests.py -k FilterMyTest
```